sharktank_models/benchmarks/conftest.py

```diff
@@ -15,6 +15,10 @@
 
 THIS_DIR = Path(__file__).parent
 sku = os.getenv("SKU", default="mi300")
+github_action_path = os.getenv("GITHUB_ACTION_PATH", str(THIS_DIR))
```

Comment:
Pass file paths in through some configuration or abstract this behind a github-actions utility script. This part of the test runner should not care what particular CI provider (if any) it is running on.